app/assets/javascripts/custom.js

```diff
@@ -57,10 +74,10 @@ $(document).ready(function(){
       };
     var url = "/cart_items/1";
     ajax_call($(this), params, url, "PATCH", function(data){
-        $('#food_sub_total_' + food_id).text(line_total);
+        $("#food_sub_total_" + food_id).text(line_total);
```

Comment:
Identifier 'food_id' is not in camel case.<br>Identifier 'line_total' is not in camel case.